src/notifications/generate.ts

```diff
@@ -94,6 +94,9 @@ export const notificationTitleMap: Record<
   collection_updated: (ctx: NotificationPostContext) =>
     `The collection "<b>${ctx.post.title}</b>" just got updated with new details`,
   dev_card_unlocked: () => 'DevCard unlocked!',
+  source_post_added: (
+    ctx: NotificationPostContext & NotificationDoneByContext,
+  ) => `New posts from <b>${ctx.source.name}</b>, check them out now!`,
```

Comment:
Oh just one minor thing, not sure who provided the copy, but it's a singular post right?
`New post from {source}, check it out now!` ?

Reply:
This is what it was in figma by Sab but yeah, I'll just do singular.

Reply:
Now that we started using GB on API we can basically load this from features also, but yes agree.